SEED-platform/seed
# Fixes bug to hide edit form on save

#### Any background context you want to provide?
From the inventory detail, after saving inventory edits the from remained in edit mode. I went back to PRs from 2023 and experienced the same behavior

#### What's this PR do?
- toggles a scope variable that hides the form.

#### How should this be manually tested?
- inventory detail 
- edit 
- make edits 
- save 
- confirm 
- 'edit' button should replace 'save' and 'cancel' buttons

#### What are the relevant tickets?

#### Screenshots (if appropriate)
